_scripts/instruction-widget/install.js

```diff
@@ -23,6 +23,7 @@ module.exports = function(context) {
 
     context.cron_included = false;
     context.dns_plugins = false;
+    context.dns_package_prefix = "";
```

Comment:
It's probably not worth worrying about right now, but it'd be nice if the build failed if we tried to generate DNS installation instructions for a system but hadn't set this. I tried just leaving it unset and building the docs hoping it'd blow up trying to use the unset variable but it didn't work.

Up to you if you want to try and figure this out (maybe by pinging someone like Vivian from E&D about it) or just make an issue for us to look into later.

Reply:
I'll spin an issue out for this.